styles/Juanda.css

```
@@ -1,3 +1,27 @@
 .section-juandavid {
-    color:rgb(104, 0, 0)
+<<<<<<< HEAD
+    color: rgb(104, 0, 0);
+=======
+    color:rgb(104, 0, 0);
+>>>>>>> d6d0c435a22b613cd640d35fd7f58ccb7fcde43a
```

Debe quedar solo la propiedad `color`
```suggestion
    color: rgb(104, 0, 0);
```